### PR TITLE
INF-21 signer service boilerplate

### PR DESCRIPTION
**Pull request by zhwrd**

### CODE REVIEW

**Comment by zhwrd** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4356** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4356" target="_blank">Graphite</a>  👈
    * **PR #4357** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4357" target="_blank">Graphite</a> 
      * **PR #4358** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4358" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4356?utm_source=stack-comment).

File: go.work [hunk 1]
```diff
@@ -19,6 +19,7 @@ use (
 	./op-proposer
 	./op-service
 	./proxyd
+	./signer
```
Comment (tynes) [inline go.work:22]: Should this be `op-signer` to keep in line with the other protocol related services?

File: signer/go.mod [hunk 2]
```diff
@@ -0,0 +1,58 @@
+module github.com/ethereum-optimism/optimism/signer
+
+go 1.18
+
+require (
+	github.com/ethereum-optimism/optimism/op-service v0.8.8
+	github.com/ethereum/go-ethereum v1.10.23
```
Comment (tynes) [inline signer/go.mod:7]: We should be on geth `v1.10.26`

File: go.work (hunk 1 above)
Comment (zhwrd) [reply go.work:22]: I considered that but didn't want to confuse people that it was part of the op-stack or something. Not sure if thats the right way to think about it but happy to change it.

File: signer/package.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@eth-optimism/signer",
```
Comment (tynes) [inline signer/package.json:2]: I thought we were going to deprecate the changeset based release flow in favor of the new release flow

Comment (zhwrd) [reply signer/package.json:2]: true, i think this is a relic from when i first started work on this over a month ago.

**Pull request closed without merging**